x-pack/legacy/plugins/siem/server/lib/detection_engine/notifications/update_notifications.test.ts

```diff
@@ -48,14 +48,13 @@ describe('updateNotifications', () => {
   it('should create a new notification if did not exist', async () => {
     (readNotifications as jest.Mock).mockResolvedValue(null);
 
-    const params = {
+    const params: UpdateNotificationParams = {
```

Comment:
NOTE: For tests when we use Typed objects rather than plain objects, TypeScript will alert us before the tests run that we need to update structures.